ruby/app/middleware/auth.rb

```diff
@@ -25,7 +25,6 @@ def call(env)
       Rack::Response.new('UNAUTHORIZED', 401, {}).finish
     end
   rescue StandardError => e
-    puts "ERR: #{e}"
```

Comment:
This change is needed because now that all tests use this middleware, the reasons why the tests fail are made too obvious through such logs. The candidate could add a log here by their own to understand why the tests are failing.